docsrc/content/abstraction-monad.fsx

```diff
@@ -219,6 +219,42 @@ module CombineWriterWithResult =
     let (_, log) = ew |> ResultT.run |> Writer.run
 
 
+// You can also stack monad transformers.
+
+// A monad transformer and a monad is itself a monad, so you can pass that into another monad transformer.
+// For example, below we are stacking them like:
+// type Example = ReaderT<DateTime, ResultT<Writer<string list, Result<string * string * string, string>>>>)
+
+// Catch and throw is generic over all monad transformers in F#+ so you can use catch works in this example
+// because there is a Result in the stack. We use it here to consolidate Result's 'TError.
```

Comment:
This sentence doesn't read quite right...

Reply:
Oops, I was logged in a work service account. Was me in above ! :)